packages/select/src/Select.tsx

```diff
@@ -14,7 +15,10 @@ import {
 } from '@ag.ds-next/core';
 import { Icon } from '@ag.ds-next/icon';
 
-type Option = { label: string; value: string; disabled?: boolean };
+export type Option = { label: string; value: string; disabled?: boolean };
+export type Options =
+	| Option[]
+	| { label: string; disabled?: boolean; options: Option[] }[];
```

Comment:
Would something like this be better?
```
type Options = {Option & {options?: Option}}[];
```

Reply:
We can't do that because when you're using an option group we don't pass in a `value`. I've just updated this PR so it reads a bit nicer